### PR TITLE
Women fashion

### PR DESCRIPTION
**Pull request by Salam-Dalloul**
### Fixes
fixes #<issue_number>

### Related
related to #<37>

### Summary
upload women cloth and price from database

### Major Changes
[**OPTIONAL for small PRs**] Detailed description of major changes in the PR, ideally with references to relevant commits, issues, line-numbers, etc.

### CODE REVIEW

File: public/js/womenFashion.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function btnClick() {
```
Comment (MahmoudMH) [inline public/js/womenFashion.js:1]: we have 2 function's do the same in menFashion .. so put one of them inside file and use it! why create the same function twice !

File: src/controllers/men.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const viewMenFa = require('../models/queries/viewMenFashion');
+
+exports.get = (req, res) => {
+  viewMenFa((dataBaseConnectionError, menFashion) => {
+    if (dataBaseConnectionError) return res.status(500).send({ error: dataBaseConnectionError });
+    return res.render('men', {
+      layout: 'fashion', menFashion, style: 'menFashion', title: 'Men Fashion',
```
Comment (MahmoudMH) [inline src/controllers/men.js]: Nice Code ! I never use it before ! so help me to understand it :+1 💯

File: src/models/queries/viewMenFashion.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const dbconnection = require('../database/db_connection');
+
+const viewMenFashion = (cb) => {
+  const viewMenFashionSql = {
+    text: 'select * from cloths where gender = \'m\'',
+  };
+  dbconnection.query(viewMenFashionSql, (dataBaseConnectionError, menFashion) => {
+    if (dataBaseConnectionError) return cb(dataBaseConnectionError);
+
+    return cb(null, menFashion.rows);
```
Comment (MahmoudMH) [inline src/models/queries/viewMenFashion.js]: i don't know why you need to return cb(...) ! 
i guess the callback return self auto ! 
Is there any error happens with you when delete return ?

File: src/views/partials/head.hbs [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <head>
   <meta charset="utf-8">
-  <title>Elegant</title>
-  <link rel="stylesheet" href="css/login.css">
+  <title>{{title}}</title>
```
Comment (MahmoudMH) [inline src/views/partials/head.hbs]: nice nice ^_^ 💃

**Review by MahmoudMH — CHANGES_REQUESTED** [review]

File: src/controllers/routes.js [hunk 5]
```diff
@@ -3,8 +3,14 @@ const express = require('express');
...
 
 const login = require('./login');
+const men = require('./men');
+const women = require('./women');
+
 
 router.get('/login', login.get);
 router.post('/loginuser', login.post);
+router.get('/men-fashion', men.get);
+router.get('/women-fashion', women.get);
```
Comment (Ghadeer93) [inline src/controllers/routes.js]: when i use this router /women-fashion , no data show . i think your database doesn't  contain test data for women-fashion . So add test data :smile:

**Review by Ghadeer93 — CHANGES_REQUESTED** [review]
adding test data for women-fashion

**Review by walaamedhat — DISMISSED** [review]
why you deleted your css pages?

File: src/views/partials/head.hbs (hunk 4 above)
Comment (walaamedhat) [reply src/views/partials/head.hbs]: I like that i will use it :+1:  good job honey

File: src/models/queries/viewMenFashion.js (hunk 3 above)
Comment (Salam-Dalloul) [reply src/models/queries/viewMenFashion.js]: it's because the function needs return at the end!

File: src/views/partials/head.hbs (hunk 4 above)
Comment (Salam-Dalloul) [reply src/views/partials/head.hbs]: :dancer:

File: src/controllers/routes.js (hunk 5 above)
Comment (Salam-Dalloul) [reply src/controllers/routes.js]: put the heroku database url in the ``` config.env ``` file, because i added the data into it :smile:

**Review by MahmoudMH — DISMISSED** [review]

**Review by muhmushtaha — CHANGES_REQUESTED** [review]
here we have the same error with the btnClick function @Salam-Dalloul

File: src/controllers/routes.js (hunk 5 above)
Comment (Ghadeer93) [reply src/controllers/routes.js]: sorry :smile:  i work as local database :+1:

**Comment by Ghadeer93** [comment]
@Salam-Dalloul  solve your conflicting files , and push it again :smile:

**Review by Ghadeer93 — CHANGES_REQUESTED** [review]
solve conflicting files

**Review by Ghadeer93 — APPROVED** [review]

**Pull request merged**